README.md

```diff
@@ -50,6 +50,7 @@ Installing on Windows
 4. double click on ```C:\Program Files\PhpStorm Protocol (Win)\run_editor.reg``` file
 5. agree to whatever Registry Editor asks you
 6. update settings at ```C:\Program Files\PhpStorm Protocol (Win)\run_editor.js``` file, because each PhpStorm version is installed into it's own sub-folder!
+    1. if you're using the JetBrains Toolbox, you're only need to set the settings.toolBoxActive flag to ```true``` and no other settings needs to be configured, because all settings will read automatically  
```

Comment:
Double check, that link to 11-14 lines in `Working under another path?` refer to correct code fragment. It's likely doesn't include new setting added in previous PR (for toolbox support).